src/System.Net.NetworkInformation/src/System.Net.NetworkInformation.csproj

```diff
@@ -228,21 +223,79 @@
+    <Compile Include="System\Net\NetworkInformation\LinuxTcpConnectionInformation.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxTcpState.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxTcpStatistics.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxUdpStatistics.cs" />
+    <Compile Include="System\Net\NetworkInformation\NetworkChange.Linux.cs" />
+    <Compile Include="System\Net\NetworkInformation\RowConfigReader.cs" />
+    <Compile Include="System\Net\NetworkInformation\IPGlobalPropertiesPal.Linux.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxIPAddressInformation.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxMulticastIPAddressInformation.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxIPGlobalProperties.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxIPGlobalStatistics.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxIPInterfaceProperties.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxIPInterfaceStatistics.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxNetworkFiles.cs" />
+    <Compile Include="System\Net\NetworkInformation\NetworkInterfacePal.Linux.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxNetworkInterface.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxIPv4InterfaceProperties.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxIPv6InterfaceProperties.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxNetDeviceFlags.cs" />
+    <Compile Include="System\Net\NetworkInformation\LinuxUnicastIPAddressInformation.cs" />
+    <Compile Include="System\Net\NetworkInformation\IPAddressCollectionImpl.cs" />
+    <!-- Linux Common -->
+    <Compile Include="$(CommonPath)\Interop\Linux\libc\Interop.sockaddr.cs">
+      <Link>Common\Interop\Linux\libc\Interop.sockaddr.cs</Link>
+    </Compile>
+    <Compile Include="$(CommonPath)\System\IO\StringParser.cs">
+      <Link>System\IO\StringParser.cs</Link>
+    </Compile>
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsOsx)' == 'true'">
+    <Compile Include="System\Net\NetworkInformation\IPGlobalPropertiesPal.Osx.cs" />
+    <Compile Include="System\Net\NetworkInformation\NetworkChange.OSX.cs" />
+    <Compile Include="System\Net\NetworkInformation\NetworkInterfacePal.Osx.cs" />
+    <Compile Include="System\Net\NetworkInformation\OsxIpInterfaceStatistics.cs" />
+    <Compile Include="System\Net\NetworkInformation\OsxIpInterfaceProperties.cs" />
+    <Compile Include="System\Net\NetworkInformation\OsxNetworkInterface.cs" />
   </ItemGroup>
-  
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Common\Interop\Unix\System.Native\" />
+  </ItemGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
-</Project>
+</Project>
```

Comment:
I prefer that as well, but it always seems futile when VS just comes and messes everything up the minute you change anything there. :disappointed:  I'll do this for the initial check-in, at least.